website/docs/r/monitor.html.markdown

```diff
@@ -528,6 +572,14 @@ Here is a summary of arguments for each condition type (fields which are not mar
       - `burn_rate_threshold` (Required): The burn rate percentage threshold.
       - `time_range` (Required): The relative time range for the burn rate percentage evaluation.  Accepted format: Optional `-` sign followed by `<number>` followed by a `<time_unit>` character: `s` for seconds, `m` for minutes, `h` for hours, `d` for days. Examples: `30m`, `-12h`.
 
+#### log_anomaly_condition
+  - `field`: The name of the field that the trigger condition will alert on. The trigger could compare the value of specified field with the threshold. If field is not specified, monitor would default to result count instead.
+  - `sensitivity`: The triggering sensitivity of the anomaly model used for this monitor.
+  - `anomaly_detector_type`: The type of anomaly model that will be used for evaluating this monitor. Possible values are: `SpikeDetector` and `Cluster`.
```

Comment:
possible values should be cluster only. spike is backward compatiblity and should not make to tf atleast.

Reply:
Removed from terraform, we should cleanup our API too.